modules/ocf_desktop/files/skel/config/xfce4/panel/launcher-20/15228073682.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Brave
+Comment=Brave laptop and desktop browser
+GenericName=Web Browser
+Exec=sh -c "brave --user-data-dir-name=brave -- %U"
```

Comment:
Is the `sh -c` necessary?

Reply:
Probably not. I can remove it, but this is the desktop file that ships with brave.